src/core/View.js

```diff
@@ -493,7 +493,7 @@ export default class View extends Component {
         const representationIds = [];
         this.selections.forEach((v) => {
           const { prop } = v.getProperties();
-          const { representationId } = prop.get('representationId');
+          const representationId = prop?.get('representationId').representationId;
```

Comment:
I'm not sure if your build pipeline supports the optional chaining operator, I can replace it with something else if not.

Reply:
Not sure either.

Reply:
It is, I just checked the `dist` output.